frontend/src/app/components/wp-relations/wp-relations-hierarchy/wp-relations-hierarchy.service.ts

```diff
@@ -136,7 +136,11 @@ export class WorkPackageRelationsHierarchyService {
         lockVersion: childWorkPackage.lockVersion
       }).then(wp => {
         if (parentWorkPackage) {
-          this.wpCacheService.require(parentWorkPackage.id!, true);
+          this.wpCacheService.loadWorkPackage(parentWorkPackage.id!, true);
+          this.halEvents.push(parentWorkPackage, {
+            eventType: 'association',
+            relatedWorkPackage: null,
```

Comment:
The halEvent `association` has an `relationType` attribute: https://github.com/opf/openproject/blob/dev/frontend/src/app/modules/hal/services/hal-events.service.ts#L25
Although it does not produce any errors now, it might in the future. So please add one here to match the interface.